nsapass

```diff
@@ -21,6 +21,8 @@ import tempfile
 
 # location of encrypted passwords db
 DB = os.path.expanduser('~/.local/share/nsapass/db.enc')
+DBLOCK = DB + '.lock'
+DBTEMP = DB + '.temp'  # should be in the same directory as DB
```

Comment:
sorry, let's also return this back to where you put them originally (inside `lock` and `save`, respectively).  your decision was better.


----
excuses for my stupidity:  one reason i thought to make this globally is because i was thinking that we may have parts of the locking mechanism in `load` and unlocking parts in `save`.  but this whole idea of mine was stupid, and what you came up with is much better (e.g. `atexit`) that also unified unlocking's definition in `lock`.

Reply:
no biggie i'll merge and do it myself.

sry i'm new to git/github.